src/neptune/constants.py

```diff
@@ -26,16 +24,13 @@
 
 """Constants used by Neptune"""
 
-ANONYMOUS = "ANONYMOUS"
```

Comment:
Does this remove the option to pass the literal string as the argument (`api_token="ANONYMOUS"`)?

Reply:
It seems so. Tried doing that and got `neptune.common.exceptions.NeptuneInvalidApiTokenException`.